mne/epochs.py

```diff
@@ -825,16 +831,26 @@ def average(self, picks=None):
 
         Notes
         -----
-        Computes an average of all epochs in the instance, even if
+        Computes an aggregate of all epochs in the instance, even if
         they correspond to different conditions. To average by condition,
         do ``epochs[condition].average()`` for each condition separately.
 
         When picks is None and epochs contain only ICA channels, no channels
         are selected, resulting in an error. This is because ICA channels
         are not considered data channels (they are of misc type) and only data
         channels are selected when picks is None.
+
+        The `method` parameter allows robust aggregation. For example, one
+        could do:
+
+            >>> from scipy.stats import trim_mean  # doctest:+SKIP
+            >>> trim = lambda x: trim_mean(x, 10, axis=0)  # doctest:+SKIP
```

Comment:
FYI moving the `lambda` out of the function you have traded one PEP8 violation (line length) for another (saving lambda as var) :)

If you want to comply you can do:
```
>>> def trim(x):
...    return trim_mean(x, 10, axis=0)
```
But I think it's okay to leave it as-is in the example.